nixos/modules/config/update-users-groups.pl

```diff
@@ -226,7 +226,7 @@ sub parseUser {
     if ($u->{createHome}) {
```

Comment:
```suggestion
    if ($u->{createHome} and !$is_dry) {
```
(and then remove the check in the line below)

Reply:
I don't mind the change, but note that this is so today and didn't touch this nor the next line.

Reply:
I've done the change.